### ISSUE TITLE
splitCssIfIeLimitIsReached adds an extra (broken) stylesheet link

### ISSUE THREAD

**Issue by salomvary** [assetgraph/assetgraph#835]
This issue has been happening since I upgraded assetgraph-builder from 5.x to 6.x.

Given a html file:

```css
<!doctype html>
<html>
<head>
  <link href="big.css" rel="stylesheet">
</head>
<body>
</body>
</html>
```

And [a big.css](https://github.com/assetgraph/assetgraph/blob/master/testdata/transforms/splitCssIfIeLimitIsReached/big.css).

The following assetgraph-builder command generates a html file that has a link to a css file that does not exist in the output:
```
buildProduction --outroot dist --root . index.html
```

Output (added line breaks for readability):
```html
<!DOCTYPE html><html><head>
<link rel=stylesheet href=static/big-0.79f1d4646d.css>
<style>body{background:url(fake.png)}</style>
<link href=big.css rel=stylesheet>
</head><body></body></html>
```

Note the extraneous `<link href=big.css rel=stylesheet>`.

```
$ npm list|grep assetgraph
├─┬ assetgraph-builder@6.0.1
│ ├─┬ assetgraph@4.0.0
│ ├─┬ assetgraph-sprite@3.0.0
```

**Comment by papandreou** [comment]
Thanks for spotting this. Fixed in 4.0.2 / ag-b 6.0.2 :)

**Comment by salomvary** [comment]
Thanks for the quick fix \o/
